src/content/docs/de/guides/imports.mdx

```diff
@@ -0,0 +1,267 @@
+---
+title: Statische Inhalte
+description: Erfahre, wie du verschiedene Inhalte in Astro importierst.
+i18nReady: true
+---
+
+Astro unterstützt die meisten statischen Elemente, ohne dass eine Konfiguration erforderlich ist. Du kannst die Anweisung `import` an beliebiger Stelle in deinem Projekt-JavaScript verwenden (einschließlich Astro-Frontmatter) und Astro wird eine optimierte Kopie dieses statischen Assets in deinem endgültigen Build einfügen. `@import` wird auch innerhalb von CSS & `<style>` Tags unterstützt.
```

Comment:
Noticing here we have “statischen Elemente” in the first sentence and then “statischen Assets” in the second — both are “static assets” in the original text. Which of these do you think is better? We should probably be consistent and pick one.

Reply:
Same problem as with `glob pattern`. Usually we use `asset` in german too, as there is no one-to-one translation with the exact same meaning in development. Nonetheless, in the context here, `Elemente` is fine, so we can go with it.